client/state/sites/seo/schema.js

```diff
@@ -0,0 +1,15 @@
+export const titleFormatSchema = {
+	type: 'object',
+	properties: {
+		id: {
```

Comment:
Let's use the pattern property here, otherwise the schema is literally looking for an object that has an `id` property. We can add a few serialize/deserialize reducer tests to make sure the schema is working as we expect.
```javascript
{
	type: 'object',
	additionalProperties: false,
	patternProperties: {
		'^\\d+$': {
			type: 'object',
			properties: {
				frontPage: { type: 'string' },
				posts: { type: 'string' },
				pages: { type: 'string' },
				groups: { type: 'string' },
				archives: { type: 'string' }
			}
		}
	}
}
```